README.md

```diff
@@ -30,10 +30,10 @@ or 2) direct installation from this repository using git.
 ### Installation from Azure Artifacts
 
 If a specific **released** BayBE version is to be installed, this can be achieved via
-[Azure Artifacts](https://***REMOVED***/_artifacts/feed/artifacts).

```

Comment:
But then tox will fail, and hence also the pipelines.

Reply:
which part of the pipeline will fail? docs is not currently in any merged pipeline. It will also automatically start working once the README PR is merged

Reply:
Ok, if docs is not part in a pipeline, then this would not fail. I'll just drop the corresponding commit for the change in the branch then.

Reply:
I removed it and force pushed. Please resolve if happy.